wallet/txauthor/author.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+// Package txauthor provides transaction creation code for wallets.
+package txauthor
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/txscript"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/btcsuite/btcutil"
+	"github.com/btcsuite/btcwallet/wallet/txrules"
+
+	h "github.com/btcsuite/btcwallet/internal/helpers"
```

Comment:
Also, any editor with "find all references" (e.g. https://github.com/redefiance/go-find-references) would show that it is a package name.  I don't see this being a real issue.

What is an issue is that this package is needed at all to reduce the verbosity of looping through the outputs just to sum various fields.